serve/mlc_serve/model/tvm_model.py

```diff
@@ -109,13 +110,17 @@ def _prepare_inputs(
         all_decode_block_tables,
         sliding_window,
         is_prefill,
+        num_decode_query_tokens,
     )
 
     if block_tables is not None:
         block_tables = tvm.nd.from_dlpack(block_tables)
     if indices_within_window is not None:
         indices_within_window = tvm.nd.from_dlpack(indices_within_window)
 
+    if not is_prefill and num_decode_query_tokens > 1:
+        input_ids = torch.reshape(input_ids, (-1, num_decode_query_tokens))
```

Comment:
@yelite Here I added logic for multi-query decoding, where the input shape is `(num_seq, num_decode_query_tokens)`.  But right now there is no decoding request type having `num_decode_query_tokens > 1`. This is left for future work in speculative decoding.